bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -557,6 +568,17 @@ private long getLastLogId(File dir) {
         return logs.get(logs.size() - 1);
     }
 
+    private long getLastLogIdInDir(File dir) {
```

Comment:
Don't we need to consider the gap? Like how it implemented in the EntryLogIdsImpl